Tests/Acceptance/DoctrineAuthCodeManagerTest.php

```diff
@@ -52,16 +52,16 @@ public function testClearExpired(): void
     private function buildClearExpiredTestData(Client $client): array
     {
         $validAuthCodes = [
-            $this->buildAuthCode('1111', '+1 day', $client),
-            $this->buildAuthCode('2222', '+1 hour', $client),
-            $this->buildAuthCode('3333', '+1 second', $client),
-            $this->buildAuthCode('4444', 'now', $client),
+            $this->buildAuthCode($client, '1111', '+1 day'),
```

Comment:
@franjo-zadelj-trikoder What is the purpose of the signature change?

Reply:
same as above